app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    <%= t(".The") %> <%= link_to t(".link1"), "http://www.railstutorial.org/" %>
```

Comment:
1. `<%= t(".The") %>`
tên biến ko dc viết hoa nhé
2. Em có thể lượt bỏ ngoặc nếu như ko có thành phần tag khác
lượt bỏ ngoặc nhớ thêm space nhé